ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.19__amd64-nightly.yaml

```diff
@@ -4444,6 +4444,77 @@ tests:
   steps:
     cluster_profile: nutanix-qe-dis
     workflow: agent-qe-nutanix-disconnected
+- as: aws-short-cert-rotation-f9
+  cron: 42 11 2,11,23 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+    test:
+    - chain: openshift-e2e-test-qe-cert-rotation
+    workflow: openshift-e2e-cert-rotation-short
+- as: gcp-short-cert-rotation-f9
+  cron: 4 22 5,14,26 * *
+  steps:
+    cluster_profile: gcp-qe
+    env:
+      FEATURE_GATES: '["ShortCertRotation=true"]'
+      FEATURE_SET: CustomNoUpgrade
+    test:
+    - chain: openshift-e2e-test-qe-cert-rotation
+    workflow: cucushift-installer-rehearse-gcp-ipi
+- as: azure-short-cert-rotation-f9
+  cron: 46 4 9,18,30 * *
+  steps:
+    cluster_profile: azure-qe
+    env:
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      FEATURE_GATES: '["ShortCertRotation=true"]'
+      FEATURE_SET: CustomNoUpgrade
+    test:
+    - chain: openshift-e2e-test-qe-cert-rotation
+    workflow: cucushift-installer-rehearse-azure-ipi
+- as: baremetal-short-cert-rotation-f9
```

Comment:
Please name this and all jobs using `workflow: cucushift-installer-rehearse-baremetalds-ipi-ovn` baremetalds to identify this job being executed with dev-scripts. These jobs aslo need to be hardcoded to execute with `cluster: build05` at this time as the ofcir provisioner step is not updated to execute successfully in other clusters.